test-resources-hibernate-reactive/test-resources-hibernate-reactive-mariadb/src/test/resources/application-jdbc.yml

```diff
@@ -0,0 +1,14 @@
+datasources:
```

Comment:
`datasources:` is not needed for Hibernate Reactive projects

Reply:
This configuration file is used in the test which shows that we can use a datasource _with_ Hibernate reactive (similar to what we do for R2DBC)